src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -295,5 +293,5 @@ enum class MappeSøkestreng(val søkestreng: String) {
     SÆRLIG_TILSYNSKREVENDE("60 Særlig tilsynskrevende"),
     SELVSTENDIG("61 Selvstendig næringsdrivende"),
     OPPLÆRING("65 Opplæring"),
-    UPLASSERT("01 Uplassert")
+    UPLASSERT("Uplassert")
```

Comment:
Ser ikke ut som det finnes noen mappe som heter Uplassert i prod:

Fra loggen så henter vi ut alle mapper for 4489 og 4483:

```Mapper funnet: Antall: 29, [MappeDto(id=100030019, navn=70 Godkjenne vedtak, enhetsnr=4489), MappeDto(id=100030018, navn=67 Saker berørt av EFTA/EØS, enhetsnr=4489), MappeDto(id=100030017, navn=66 Tilbakekreving infotrygd – Klar til behandling, enhetsnr=4489), MappeDto(id=100030016, navn=65 Opplæring, enhetsnr=4489), MappeDto(id=100029732, navn=EF Sak - 65 Opplæring, enhetsnr=4489), MappeDto(id=100028592, navn=EF Sak - 70 Godkjenne vedtak, enhetsnr=4489), MappeDto(id=100028584, navn=EF Sak - 64 Utdanning, enhetsnr=4489), MappeDto(id=100028583, navn=EF Sak - 63 Inntektskontroll, enhetsnr=4489), MappeDto(id=100028582, navn=EF Sak - 62 Hendelser, enhetsnr=4489), MappeDto(id=100028581, navn=EF Sak - 61 Selvstendig næringsdrivende, enhetsnr=4489), MappeDto(id=100028580, navn=EF Sak - 60 Særlig tilsynskrevende, enhetsnr=4489), MappeDto(id=100028579, navn=EF Sak - 50 Tilbakekreving - Klar til behandling, enhetsnr=4489), MappeDto(id=100028578, navn=EF Sak - 41 Revurdering, enhetsnr=4489), MappeDto(id=100028577, navn=EF Sak - 40 Revurdering - Klar til behandling, enhetsnr=4489), MappeDto(id=100028576, navn=EF Sak - 30 Klager - Klar til behandling, enhetsnr=4489), MappeDto(id=100028575, navn=EF Sak - 20 Avventer dokumentasjon, enhetsnr=4489), MappeDto(id=100028574, navn=EF Sak - 10 Søknader - Klar til behandling, enhetsnr=4489), MappeDto(id=100028573, navn=EF Sak - 01 Uplassert, enhetsnr=4489), MappeDto(id=100026882, navn=64 Utdanning, enhetsnr=4489), MappeDto(id=100026881, navn=63 Inntektskontroll, enhetsnr=4489), MappeDto(id=100026880, navn=62 Hendelser, enhetsnr=4489), MappeDto(id=100026879, navn=61 Selvstendig næringsdrivende, enhetsnr=4489), MappeDto(id=100026878, navn=50 Tilbakekreving - Klar til behandling, enhetsnr=4489), MappeDto(id=100026877, navn=41 Revurdering, enhetsnr=4489), MappeDto(id=100026876, navn=40 Revurdering - Klar til behandling, enhetsnr=4489), MappeDto(id=100026875, navn=10 Søknader - Klar til behandling, enhetsnr=4489), MappeDto(id=100026874, navn=30 Klager - Klar til behandling, enhetsnr=4489), MappeDto(id=100026873, navn=20 Avventer dokumentasjon, enhetsnr=4489), MappeDto(id=100026872, navn=60 Særlig tilsynskrevende, enhetsnr=4489)] ```


og 

```
Mapper funnet: Antall: 29, [MappeDto(id=100030015, navn=PENSJON - Etteroppgjør, enhetsnr=4483), MappeDto(id=100029584, navn=PENSJON - Utland - Klar til behandling, enhetsnr=4483), MappeDto(id=100029360, navn=PENSJON - Åpenbar sak/omgjøring klager/kurantsaker, enhetsnr=4483), MappeDto(id=100029359, navn=PENSJON - Tps-bt-ft, enhetsnr=4483), MappeDto(id=100029358, navn=PENSJON - 50 Tilbakekreving - Klar til behandling, enhetsnr=4483), MappeDto(id=100029357, navn=PENSJON - 41 Revurdering, enhetsnr=4483), MappeDto(id=100029356, navn=PENSJON - 40 Revurdering- Klar til behandling, enhetsnr=4483), MappeDto(id=100029355, navn=PENSJON - 30 Klager- Klar til behandling, enhetsnr=4483), MappeDto(id=100029354, navn=PENSJON - 20 Avventer dokumentasjon, enhetsnr=4483), MappeDto(id=100029353, navn=PENSJON - 10 Søknader- Klar til behandling, enhetsnr=4483), MappeDto(id=100028946, navn=EF Sak - 70 Godkjenne vedtak, enhetsnr=4483), MappeDto(id=100028945, navn=EF Sak - 64 Utdanning, enhetsnr=4483), MappeDto(id=100028944, navn=EF Sak - 63 Inntektskontroll, enhetsnr=4483), MappeDto(id=100028943, navn=EF Sak - 62 Hendelser, enhetsnr=4483), MappeDto(id=100028942, navn=EF Sak - 61 Selvstendig næringsdrivende, enhetsnr=4483), MappeDto(id=100028941, navn=EF Sak - 60 Særlig tilsynskrevende, enhetsnr=4483), MappeDto(id=100028940, navn=EF Sak - 50 Tilbakekreving - Klar til behandling, enhetsnr=4483), MappeDto(id=100028939, navn=EF Sak - 41 Revurdering, enhetsnr=4483), MappeDto(id=100028938, navn=EF Sak - 40 Revurdering - Klar til behandling, enhetsnr=4483), MappeDto(id=100028937, navn=EF Sak - 30 Klager - Klar til behandling, enhetsnr=4483), MappeDto(id=100028936, navn=EF Sak - 20 Avventer dokumentasjon, enhetsnr=4483), MappeDto(id=100028935, navn=EF Sak - 10 Søknader - Klar til behandling, enhetsnr=4483), MappeDto(id=100028934, navn=EF Sak - 01 Uplassert, enhetsnr=4483), MappeDto(id=100025583, navn=50 Tilbakekreving - Klar til behandling, enhetsnr=4483), MappeDto(id=100025582, navn=41 Revurdering, enhetsnr=4483), MappeDto(id=100025581, navn=40 Revurdering- Klar til behandling, enhetsnr=4483), MappeDto(id=100025580, navn=30 Klager- Klar til behandling, enhetsnr=4483), MappeDto(id=100025579, navn=20 Avventer dokumentasjon, enhetsnr=4483), MappeDto(id=100025578, navn=10 Søknader- Klar til behandling, enhetsnr=4483)]
  ```

Reply:
Ah, ok, det fantes ikke en tilsvarende uplassert mappe der. Er det greit å sette mappeId lik null for uplasserte mapper ?